packages/core/src/svg/some/whatsapp.svg

```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 24 24" version="1.1" role="img" xmlns="http://www.w3.org/2000/svg">
```

Comment:
It probably makes zero difference but this specifies `version="1.1"` which is not currently included in other SVG icons.

Reply:
I think it can be removed since it is deprecated and it has no effect on rendering or processing: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/version

Reply:
- [ ] Remove `version=1.1`